### DISCUSSION TITLE
Selecting developers for my project

### DISCUSSION THREAD

**Discussion by Suandrip** [community/community#147263] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello everyone. I am trying to build a Python-based trading system on an AWS server. I know, this may sound silly, but does anyone have any tips on how I select a reliable developer to help me with the project? For starters, they should be well-familiar with Python libraries and JavaScript.

**Comment by Logophoman** [comment]
Can you give more background information?

1. What do you want to build exactly (just rough outline, what is traded, is it real-time etc...)
2. Why do you think it should be python in the first place? Why not another technology stack?
3. Do you have a budget, are you looking for hiring an employee or more like someone from Fiverr? Since that will determine what you want to be after...
4. How big is the scope of the project and what is to be done?

**Reply by Suandrip** [reply]
1. I am looking to start with cryptocurrency trading (more specifically, integrating Binance API with Python). But I will expand to other assets (forex, stocks) in due course. I want to build an automated trading system with a tick-based chart for visualization purposes. I am looking to construct a trading system that sends orders at specific time intervals. 
2. I believe Python is versatile and greatly customizable. I have experience designing EAs for the Meta Trader platform, but I am new to this API based integration (although DLLs seem similar). This project is mainly a proof of concept, there will be trial-and-error based methodology.
3. Well, I am initially trying to keep it within 3000 $. I am looking for more of a guide, a partner maybe, not an employee.
4. The idea is to build an automated trading system with Python for the back end and JavaScript for the front. We draw the instrument data from the server, plot a tick-based chart (10-tick interval), find the median value and then find the direction of the candle (bullish or bearish). Then we place a post-only order (at the median price) in the opposite direction, say sell for bullish and buy for bearish. The orders are grouped together with SL and TP during creation. That's it. No complex indicators, or statistical analysis. But I do think it will consume humongous computing resources.

I am open to suggestions. Feel free to seek any further clarifications.
